sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -591,6 +591,7 @@ public static Read read() {
         .setParseFn(parseFn)
         .setMethod(Method.DEFAULT)
         .setUseAvroLogicalTypes(false)
+        .setFormat(DataFormat.DATA_FORMAT_UNSPECIFIED)
```

Comment:
```suggestion
        .setFormat(DataFormat.AVRO)
```

The default should be specified here, instead of relying on execution time code.

Reply:
Changes made